app/src/test/java/org/simple/clinic/summary/PatientSummaryUpdateTest.kt

```diff
@@ -655,6 +655,27 @@ class PatientSummaryUpdateTest {
         )
   }
 
+  @Test
+  fun `when contact doctor button is clicked and more than one phone number is present, then open contact doctor sheet`() {
```

Comment:
Instead of describing the implementation detail only one phone number is present, can we reword this to reflect the real world usage? Ex: `multiple doctors are available for teleconsultation`.